docassemble/VirtualCourtSampleInterviews/data/questions/Text Date Input.yml

```diff
@@ -24,9 +24,6 @@ fields:
   - Date of birth: date_of_birth
     datatype: textdate    
     default: '2/1/2000'
-    validate: |
-      lambda date_of_birth: True if is_date_valid( date_of_birth ).valid else validation_error("This date doesn't look right")
```

Comment:
Strange, this works for me without it. In fact, you reminded me that I should delete the module reference, so I did that and it still worked (will need to check it in). I believe this is because it inherits the server side validation, as Jonathan told me when I was working on it. In addition, at one point we agreed that using lambda to validate for every instance is cumbersome in practice.